lib/utils.coffee

```diff
@@ -292,10 +293,12 @@ timeoutPromise = (ms, promise) ->
 # utils.requestAsync({ url: 'http://example.com' }).then (response) ->
 # 	console.log(response)
 ###
-exports.requestAsync = (options) ->
+exports.requestAsync = (options, retriesRemaining = undefined) ->
```

Comment:
This `retriesRemaining = undefined` is redundant, it will only be set to `undefined` if it is already nullish, but then `retriesRemaining ?= opts.retries` will also set it if it's nullish, making it redundant to explicitly change nulls into undefineds